src/Agent/NewRelic.Api.Agent/NewRelic.Api.Agent.csproj

```diff
@@ -7,6 +7,8 @@
     <DocumentationFile>$(RootProjectDirectory)\src\_build\$(Platform)-$(Configuration)\$(AssemblyName)\NewRelic.Api.Agent.xml</DocumentationFile>
     <IntermediateOutputPath>$(RootProjectDirectory)\src\_build\$(Platform)-$(Configuration)\$(AssemblyName)\</IntermediateOutputPath>
     <OutputPath>$(RootProjectDirectory)\src\_build\$(Platform)-$(Configuration)\$(AssemblyName)\</OutputPath>
+    <Nullable>enable</Nullable>
+    <LangVersion>8.0</LangVersion>
```

Comment:
I believe we can declare `LangVersion` in one spot. Change this to 8.0 here:

https://github.com/newrelic/newrelic-dotnet-agent/blob/0182016ebc837f56dc61829e99abe67633e38c95/Directory.Build.props#L3

Then you should be able to remove `LangVersion` from individual csproj files.